pkg/kubelet/status/status_manager.go

```diff
@@ -261,28 +266,32 @@ func (m *manager) SetContainerStartup(podUID types.UID, containerID kubecontaine
 
 	pod, ok := m.podManager.GetPodByUID(podUID)
 	if !ok {
-		klog.V(4).Infof("Pod %q has been deleted, no need to update startup", string(podUID))
+		klog.V(4).InfoS("Pod has been deleted, no need to update startup", "pod", string(podUID))
```

Comment:
```suggestion
		klog.V(4).InfoS("Pod has been deleted, no need to update startup", "podUID", string(podUID))
```

Reply:
ok, I misread the https://github.com/kubernetes/kubernetes/pull/99836#discussion_r588257305 fixing it..